mantidimaging/gui/windows/main/view.py

```diff
@@ -474,13 +474,10 @@ def live_view_choose_directory(self) -> None:
             self.show_live_viewer(Path(live_data_directory))
 
     def show_live_viewer(self, live_data_path: Path) -> None:
-        if not self.live_viewer:
-            self.live_viewer = LiveViewerWindowView(self, live_data_path)
-            self.live_viewer.show()
-        else:
-            self.live_viewer.activateWindow()
-            self.live_viewer.raise_()
-            self.live_viewer.show()
+        self.live_viewer = LiveViewerWindowView(self, live_data_path)
+        self.live_viewer.activateWindow()
+        self.live_viewer.raise_()
+        self.live_viewer.show()
```

Comment:
This will need to be made into a list of liveviewers, so that `closeEvent()` can close all of them. Otherwise there are issues if you close the main window with multiple live viewers open.

Also in `LiveViewerWindowView.closeEvent()` it will need to remove itself from this list.